services/blockchain-service/blockchain-service-base.js

```diff
@@ -1204,4 +1206,37 @@ export default class BlockchainServiceBase {
     convertToWei(ether) {
         return Web3.utils.toWei(ether.toString(), 'ether');
     }
+
+    //Paymaster functions
+    async deployPaymasterContractFunction(blockchain, hubAddress) {
+
+        return this.callContractFunction('Paymaster', 'constructor', [hubAddress], blockchain);
```

Comment:
This function is not in Paymaster but in PaymasterManager, check dkg-evm-module.

It also doesn't accept any arguments

Reply:
For this PaymasterManager ABI needs to be imported